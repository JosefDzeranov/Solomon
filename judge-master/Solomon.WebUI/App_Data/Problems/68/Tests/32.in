934 7
-21862 -80186 78632 -23602
605 24812 22610 55046
-57431 -25187 -53453 -8001
-66854 1898 77793 82585
-17137 -49693 24783 -6117
-63186 -61746 1231 -42765
-31553 15849 -10770 42427
-62247 -52839 91617 -24138
9530 66893 98398 93052
-86401 -25055 -34305 8977
-53574 18528 84700 60101
-38979 -73186 10575 -66698
43500 4554 48163 44912
-94373 28977 -4719 60236
38929 -97973 71622 38290
-17342 11710 26657 25754
70113 -75935 88043 93546
-51280 -41897 35778 -25148
-24971 -53804 -16930 33331
-92320 -82570 62393 -57379
-93491 26885 18364 59019
-98759 31670 -69605 80290
-85469 -17752 -32772 22230
5271 -21683 61557 77075
44316 -22490 94707 69986
-28119 21867 74480 27663
-44987 23185 48933 51850
-42088 48346 61582 49633
-56365 -71056 36868 -43093
59980 -29487 82871 21315
9348 -54425 77047 23456
14075 -90271 99965 78252
-28183 32824 98900 83042
91749 -58092 94814 -13368
-95039 51438 55967 65320
-15405 -48746 55954 -341
-75071 29782 88296 42523
-71556 28937 -41768 78717
16083 -21760 24542 19407
-33734 63775 94201 91716
-36949 -17729 -30039 -71
27411 -17204 56198 -5913
-75847 -96605 15327 -19136
-54693 -35895 -25538 3411
-5095 31362 13878 88442
-84141 -64110 26538 -8329
-75083 -93321 -15485 26109
-99315 -60007 48303 -50247
-59928 -52549 52741 -23931
32281 -85431 44238 -57794
-88426 -48824 -41231 -28872
-69412 -49234 -56272 70674
-75144 -13461 -75081 76882
20668 -52414 63911 34573
-16331 10800 36828 29492
-59338 20845 83381 37622
47460 -845 89051 7573
-69428 -23051 30855 -7260
-26341 -82191 -22898 45647
41487 -87405 45834 35525
27942 -5019 48918 88228
-40409 -87533 -16710 -48640
302 20389 42016 88128
-40588 -97928 -27596 25564
-32773 9434 -20766 94245
36209 9862 81831 91437
-82417 -27835 -79087 55923
-67459 27171 76893 74919
-94147 8503 15509 75231
-24289 -78679 -23007 92477
23363 24010 53466 35913
-20056 36494 6823 60964
-95538 -94045 -13152 72042
-92349 -96770 -75391 -82755
20199 -72979 91059 -51272
-55992 8127 -39156 82622
-90493 24553 -49050 32892
6728 -43579 72879 39371
-42966 -89004 63187 -59066
83689 -37409 85506 17207
-84090 -17990 -47411 -5815
-73 59598 8904 89593
-49627 -43625 -16645 38784
11559 18243 46363 75691
-23449 -59013 2568 -12783
44386 -71169 82701 74806
-35095 -82501 64696 -37788
95381 41861 96768 75971
-37626 -63435 2952 -30667
-32785 -39570 22165 -33260
-60797 -63489 -39254 -1081
-38074 -71620 -32850 77842
-66560 -41437 -9253 13672
-59220 -99677 23598 -6333
-51120 -65000 -12179 -12976
-42741 -26834 -3160 74160
-35568 -69727 33259 -56992
-36409 -86893 35475 -48551
-95102 51302 -86943 54729
-23390 -33433 -16643 59101
46379 39602 76680 84042
-57610 37774 -25729 98631
-70538 -98671 2213 -27338
72767 -92592 80398 60690
-71228 -92574 -25667 -56997
36342 5070 54286 84265
-92020 -3383 -78176 -231
-60753 -75827 -4485 -5825
-96016 -70995 23155 -34616
-65120 67289 74218 67478
46440 34944 75621 91650
44709 -9750 74984 89034
48327 -49031 55273 -38819
-79342 -89361 -63088 -26590
-87879 -51230 -23140 49157
-50217 -33543 28940 -9536
-78772 -25779 -35443 45135
-85541 22541 -76787 77888
-56052 -52272 -5986 -47126
26444 79118 53507 97262
-13656 -25397 94170 -20741
-77953 -90039 -19699 -22054
31258 16940 59196 24878
-72107 14324 47479 41171
47237 -25533 64212 27827
-50060 -88852 -27050 59884
-38458 -53061 -29363 67686
-66361 -87959 21619 -38933
-98882 -42312 -20393 49474
-87592 -56298 99732 -38516
-79245 -99887 64498 -52969
-63364 39465 45266 42360
-1843 -41753 98393 -30235
-95387 68134 15941 80550
58312 -44645 61298 13203
-93226 -90344 -59220 -56386
-38636 6416 74685 52952
-54850 -87916 36688 -53334
-42959 -30231 -28972 25469
-84602 -93680 25737 -407
-88323 7836 40794 40053
-74702 -51385 7219 -22712
-94860 -84916 -19821 -25389
7774 -81109 85529 -49033
-91998 23169 56972 85555
-63022 -30041 -14199 56241
-1375 30867 16844 72567
7954 -50581 98380 -6157
-34535 40035 22408 51723
-91202 -47849 -23581 91423
-17576 47222 8239 84751
-55569 -77028 -8176 -75040
74780 -40293 90211 83214
-44325 -80911 10651 -52891
52270 -56344 94920 30486
-54564 -54174 -24492 11201
51940 -45275 66055 41477
-27579 -61857 -12589 15539
-33993 -4183 -31811 87636
66163 -82831 68757 -75329
37499 -3121 58166 73669
-76468 -97802 -22684 72672
60928 -53053 82317 80433
8026 -83307 33102 -24189
8624 78734 64370 91859
-67069 -39857 -4699 39010
-96422 44911 50082 49644
-95603 -32454 24755 -8037
-46928 28165 98421 42305
-88868 -40314 -34694 70334
-79511 -91354 -27657 99734
42272 -11928 42322 57069
-83441 33820 -49417 61428
16079 -33364 32061 -3766
-35289 58880 88585 79426
-56308 -55997 87727 -9604
1826 58541 23134 85675
-61180 -58587 -22968 72750
-54575 -14816 49193 -13904
-93432 18075 26135 25538
-47348 42423 74144 80847
-77962 -37002 64754 -12180
22621 23467 39514 38501
18133 -63280 80852 -7202
-27055 -60722 51645 -56725
-64826 -74812 87369 -67976
-79080 -64401 71071 -648
72106 -26774 90769 93350
32989 34952 61688 99898
53777 -31812 94069 65620
-51507 -39107 -39127 61315
-98549 -37841 -69273 2577
48982 -5069 99020 30701
43396 -14198 93304 21919
-18899 -74613 76721 -6399
-6411 10758 51903 33377
-94336 61245 -38821 75605
10108 1053 99602 70608
34229 -80717 42741 50145
6939 9308 23646 51640
-44997 24718 99540 79122
-96197 2893 -40378 51635
466 6783 91697 88068
-66643 8244 421 31967
-62049 2173 -35300 92757
41307 -65397 72064 22174
-72423 -41896 -39470 50288
-39060 -99824 13085 -49951
-73583 -19546 -39576 -3542
-88238 -84445 -64813 -64788
17322 48605 26748 58204
-40749 -66490 -2034 79816
48188 82555 69736 98452
-56743 57750 7270 87969
25440 14142 28031 85623
9069 36676 43055 82374
-30295 6082 1034 92399
76702 -53288 89455 59570
33674 -97650 77370 -28709
15426 -39014 27160 8345
83805 8249 94528 14012
6004 -91355 84675 -49669
-62089 -39807 -44633 41630
58564 -57355 66444 88863
-58400 12143 13156 85023
-14634 45260 95528 51745
33242 -56061 44854 50006
-60888 -90014 -32948 -22035
-86927 -12578 -10527 74299
-52385 42199 30069 89975
-81069 -1921 -51381 24730
-11154 26176 41595 34465
54843 -76182 63097 -61281
-69384 -99153 -19231 -66304
11605 -83733 62725 31260
-12266 -50082 80565 -21161
3366 -78476 45142 59935
53085 -37281 87105 94495
22721 -33398 82846 82479
-19953 -38039 -6848 51582
-37320 -99708 16899 -83475
-35470 -40122 -14441 -27977
-63517 21412 -13645 73600
16151 -89458 56922 -38782
-38679 -51364 -10017 59803
-3857 -14660 90962 -781
-46199 16904 -35262 96869
7780 -4191 66441 45736
22159 -43234 35653 -37096
38318 19965 60946 23346
29813 -96890 44542 -93394
63083 -78985 87122 14766
51418 -91824 85432 -42249
46924 -34818 59960 81044
-68180 -43571 -54089 32570
-22561 -83926 98998 -39885
-88191 -59136 -969 77780
10641 -93225 16152 -30115
-15180 66528 41879 81274
-88399 -88657 -171 -18034
46258 20761 66018 97567
-70969 -54269 15937 -46748
-12695 -72500 -6269 -37877
32776 -76348 64231 -70770
496 -1636 80116 76381
-99849 -39378 -7532 3208
32579 -41765 82420 3841
31914 -74311 98014 72434
-58216 50683 71991 90038
-62352 -93484 -32851 -76737
-73031 -7041 -16979 63208
32466 -37490 85037 19589
63838 -72712 73080 15675
-77041 -66668 -68758 84577
9602 44707 96147 74845
-48041 47961 682 88001
-40404 -77419 -21871 92863
8443 5867 16836 25611
22129 10873 91993 94130
48255 -37070 97647 97858
11934 -92804 63339 57191
-16278 -96016 39296 -5976
-11479 23869 24495 37854
30944 -40487 44186 -13937
-10372 -59566 73961 -51650
-61050 -95661 -10810 -70349
5647 -40496 88322 94412
-70155 -48514 -41608 92210
-7067 -83872 72277 -75215
55057 -63691 64846 77088
-93663 -88986 -72250 -47310
-29775 -73500 -15518 -13931
37286 -20820 70729 66653
-57569 45983 40478 47916
-72941 1728 -52834 1803
-10260 70566 44493 87370
-34238 -36888 -23139 2778
7414 -85897 66214 30660
44672 -80214 71386 -34948
-64938 -28443 -56958 18257
64639 -85377 71793 50977
-21281 37432 63549 60006
28749 -55901 74500 55254
24295 -4478 32855 68867
23402 92771 83003 94632
-99334 -89338 -83310 -3939
31003 24540 65320 98366
-57568 -84048 -49717 -31034
-78972 -31006 -52880 1717
22440 28780 73709 58511
-20323 55137 41326 95015
60504 -3907 77599 34464
9373 55594 11052 77294
54912 -19041 73492 40301
50394 -11242 86185 54900
17028 -67177 58103 29042
96179 -79358 98633 23602
26091 -77561 33327 84283
-93607 15872 -83748 71055
-54579 -50484 81368 -16325
56868 -98708 81706 25827
-97226 273 -40284 77752
38774 -57836 73417 97924
17791 -67711 76040 34818
-53211 -63778 10167 -58627
78699 985 85777 91148
89613 17197 96653 54153
47635 -24521 77947 91133
49418 -58374 82791 39231
565 -50127 35101 -24193
-61919 -76603 -23939 61685
-56329 85630 55812 93452
-10143 62544 12086 65410
90487 -78847 96538 -61443
12024 -75235 54047 -59599
-63119 -97390 -61118 36927
-91542 -48104 12072 -20748
-69125 -19923 -32362 -14547
6485 -99305 44448 68633
-55919 -41718 35070 -24043
-81254 17055 48933 37316
-8569 15603 76503 36565
20028 28462 21036 43291
-72462 -81411 -9706 36161
-39198 3099 8682 12197
-75433 -77416 95412 -57867
18116 -96596 86327 73108
66962 -89093 74652 -23744
87786 -68521 88817 17998
-32237 15113 47453 27994
-82647 11311 51954 25088
14714 -33184 49399 41329
-69361 43568 -63956 71265
-97767 17450 -92083 57542
-48939 -96080 69649 -36353
-7471 29011 46035 59584
-65350 -36155 -4317 -717
73745 -6513 88906 3121
-99278 -72537 -32076 71213
-91277 -38621 -31629 -3260
23843 -93111 88695 13199
-55745 -56216 -53632 -23270
-43356 -81425 -1738 -70798
-1405 48117 36039 91893
91176 3302 94963 64560
19802 -17024 19908 81824
-78045 -31055 -65763 -17990
-36167 1963 89678 62850
-97385 -59008 -13739 -3214
-29260 -97290 86247 -524
42907 -40213 77022 63260
-69095 22662 81443 64434
9543 10421 31267 40626
-52572 50572 -11990 62757
20980 -74935 69707 49845
-72253 11657 99445 87833
-82332 -61831 -64690 -61801
-81149 17729 95946 93176
-57841 -86529 22706 -18358
-22138 -52568 -10546 79807
23542 37980 37843 62674
-37250 26716 -2623 92139
-53518 44061 -32678 90032
-80117 6607 81383 82934
-8438 -89382 7377 -27760
14495 -55271 34389 -30771
-70990 -65788 -18490 -60262
10425 -89542 68215 76832
-93225 -8780 -60568 52785
28374 13567 48414 95877
-58915 -58214 -27841 -22260
-67684 36147 -27131 63136
-67287 -93196 -2485 45043
-3054 34869 62807 60189
-33392 14585 -1983 86722
-44265 -99482 42387 -10045
-39941 -68617 -27896 43816
22123 -59656 53771 -31659
80020 -95584 99557 -86902
-24462 -22319 -13337 -8186
-44300 11007 63248 41680
-53981 -83577 69578 -2547
-85240 54545 22681 99090
73631 -85790 85932 60901
-49056 -50022 90412 -4366
86534 -88510 97805 3851
35336 -95037 76155 -16187
-49756 15434 41218 52658
-31204 -44615 -2527 31695
-54565 3842 -36206 64673
60115 -98408 81920 -78590
30227 -23392 64861 43875
-77637 -50004 -16866 -39077
-77893 -54053 53448 -13710
-91732 -63845 17503 -3281
-39610 19830 60602 98840
-95648 8246 -93018 84900
21880 -94670 26335 -28107
-1237 18264 4609 94916
61426 -84641 66730 -2672
19896 -23238 82639 -3023
37863 -88440 98178 4514
-87947 61570 -83198 84014
-48456 -22364 -37884 46141
32549 -12826 94295 33715
-76406 -86316 82461 -70464
-54087 -52776 12887 -28773
57490 87041 87179 94974
-30213 -18488 96639 -13082
-57714 7016 77351 47460
53148 -93954 86570 1076
10966 -24470 28198 4474
-28247 -30465 40989 -21259
-16454 32092 70808 67188
-59666 1414 24202 65764
-7366 3902 49182 90251
34174 -17408 74486 14053
-58191 -8745 -3879 -8724
46465 -62705 95437 -40761
-30730 -88423 80584 -87535
-24266 40999 -15652 83434
-24780 15147 -3573 15332
-97109 -61618 36032 -41749
74041 33985 76800 63984
-69807 -59190 -25927 19500
-48144 -82 -16547 97583
-50949 33990 -45674 93358
53512 -92373 73642 -25478
-30801 13503 -26066 36806
-94893 35453 20189 77814
-33837 -45078 -11543 77580
75973 -97780 94638 78095
59211 4527 89257 99473
-48942 -98062 -40177 39608
-26039 -75109 6254 -58635
-59736 -54703 -38020 88085
34275 44450 87148 94539
33213 -8595 49028 32205
-25995 55025 -6920 89868
41279 -73744 91567 -32882
-98377 -35380 -37447 13590
10158 -65961 35125 -31833
-91996 -71976 -84582 41291
-48359 -93410 28402 -18184
48072 -85229 57661 18217
-37214 58910 7130 89822
-70501 -20839 -23955 40103
-81910 4732 -52413 82977
-96256 -29813 -1115 8161
-28069 -23313 94882 -6269
-85682 -6240 -31595 41312
6893 47955 58716 57152
8078 -59424 93192 84549
-62383 52728 -44231 58291
26756 -85777 84518 23656
-9314 -17536 36312 -4590
-86017 -21868 -5943 54942
20876 -4888 28513 42238
-81437 -20742 -75763 84823
78880 -97427 98049 32122
-18029 -69076 76930 -65139
-60883 -93916 -17066 -63716
55676 -41438 89045 18242
-94295 -19389 -63403 -9919
-96205 -46726 -40275 -19107
47596 -79540 60788 -36767
30364 -58413 36762 28916
-26460 -73620 72770 -60699
75890 -50028 94812 -4986
-21717 -68756 61188 -27769
-27724 -60507 67282 -29209
-17814 -81078 -8424 -70036
-67661 19673 -41153 46969
-70294 24766 31522 67681
-73706 -72426 -49355 -13191
-80743 76869 -13962 94320
-20349 -43023 41456 -37809
-92222 8077 -89284 99561
-40520 -45194 49211 -6002
-6931 -19932 84413 -11470
-56948 7429 -4159 57593
-73266 -23351 -57709 2020
-75455 -58280 58458 -48146
-67493 -55251 -64697 43857
-58880 -73354 39896 -35204
24838 -79385 26324 -30735
-91492 14135 96692 90135
-71243 -82305 11305 -68862
-65783 -74368 87238 -53967
46453 4958 84327 46460
-95447 -32851 -10267 -32197
-82200 76543 -9800 93651
-72011 -92905 -20742 -28715
-75965 92824 12317 95075
-95522 -77944 -23544 61347
18102 -21977 82472 12301
-81272 32350 -14614 52788
31194 -11528 58744 28166
35977 44236 73818 99399
7533 30493 12329 68566
73688 -57518 74945 60767
-99377 32260 -61619 65897
52021 -20600 71131 87086
11454 -43388 92313 99165
87849 -78275 95985 50196
-50312 -39850 -24933 21890
44591 -81759 79706 82579
-9552 -76348 88686 -23331
2975 70306 9580 87602
-14657 -27606 -14523 78526
-90213 -53649 -20369 10928
-38501 7087 29814 9924
-89904 -57051 -71067 86834
-84163 -44799 -36526 -42079
-87282 -45907 -37643 -33616
-47274 -90660 -5487 -46080
-53598 -95645 -51102 70889
-55396 -11601 -10243 18636
-19783 -76332 500 -11833
23462 92398 51169 93898
-33605 25930 -18234 87146
-4211 -35763 4011 -74
-84701 -95686 79870 -70538
-924 36263 31132 90641
-5936 44629 97073 53570
-93429 -39351 -25870 -8290
50815 -28227 99602 62372
-74524 23651 11024 84490
25360 -16166 77988 81844
13870 -22764 78231 -9487
-97088 41178 27875 98340
8925 -51577 34956 34765
-62942 -44382 53861 -37560
14591 -42175 31864 13491
-29732 -59937 6668 -48769
-44654 88845 32760 90825
64171 -64976 99380 46568
34136 -95468 64644 34977
-35276 -26066 67833 -2170
-80985 34409 27059 57887
-98386 20818 -59647 93513
-50973 -71266 85442 -44801
31955 60913 36521 82953
21338 -75623 49644 99619
-97724 -87627 -75651 -78560
-78485 -93869 -16082 -78431
-66204 53194 -5054 77467
15134 -52257 40136 6067
-31255 -34901 90008 -11028
-34383 -71910 -28643 52829
-6949 4952 52238 63611
-15187 -80423 91820 -71634
-89497 29080 21915 72641
-61381 -16718 -1347 51257
19399 40707 71271 58282
6032 55444 81947 64598
21130 75983 64564 87727
-74830 614 56342 89133
-25582 -63911 68608 -46171
-71075 -69660 53511 -32293
-87181 -54989 -20083 -14338
-63325 49870 17531 62700
-94372 38069 17398 59041
-73288 -19707 -35980 66381
-84365 -5244 -11912 -799
-24986 -96894 -18218 30156
-84963 70036 -47865 80093
48092 25912 68871 76306
-33588 -45647 -9045 38214
5113 30053 41710 42039
73222 -26013 98036 53005
-63876 1207 94054 78139
17715 12080 75822 35159
-82957 -34206 58685 -818
-96968 58134 14573 95744
-22651 -41231 49632 -39021
36236 -11115 44432 35376
31252 -61916 89199 37226
3249 88176 39151 89972
-41897 38703 -21335 57747
79603 -94450 96007 37797
-36240 42500 -29428 88897
-81060 72557 -30078 82327
-64538 -36336 96073 -2645
20927 -97190 30032 -2070
-61764 18883 -14523 52989
13310 -86665 43767 -8993
-80795 -19696 -13691 54276
-42747 36872 15609 80581
15277 -77067 52228 29849
-22553 -35556 -4180 -8312
2244 16410 48827 38092
10549 44539 38168 59592
-82031 -58517 18480 -13851
865 -57469 50448 77037
1997 -42707 65489 75307
-97492 -62146 58249 -33472
-6796 -86554 -781 32212
-26775 11836 34624 35118
-78635 -40174 -14414 39854
17851 12381 76748 41018
-24586 -93889 -19235 -85334
-57594 38356 -55561 63865
-92384 -28516 -36638 22207
67886 -50903 90974 44160
25918 -79047 79189 -38045
-30034 35542 10468 48342
-86335 -92748 -3586 -71599
-29599 8515 78215 26322
-52491 -81761 62262 -69218
3000 -77705 45235 -64147
-75181 -65217 -42821 -59951
-21833 17076 21152 23614
42222 -14034 43135 73839
48641 90345 61031 98949
45749 -78672 49342 -70553
-35792 86587 42003 99098
25898 -10821 75038 83058
9999 -96617 34635 46330
-98212 -34992 -78751 7503
-22509 43967 66204 47549
-73713 -71079 -67619 69637
-32542 17696 96211 98376
40619 -33512 47271 82009
-98071 -97090 -7678 -18523
-97799 37329 -52617 38056
-80315 68113 4560 86459
-19721 -54902 20801 -15519
17050 1835 67456 40605
-27796 -90156 95444 -32272
-38873 -86576 -23963 -62001
29261 42395 74625 53026
-59223 48317 -48752 62716
-41924 -81398 3987 -4676
-57621 12056 83163 42684
16451 -33959 48273 72311
-44732 18621 -36774 65729
-85076 37619 -46936 72933
-62354 -79577 -31012 99295
-63939 78132 81078 91303
5399 13282 67853 16300
-19641 -33754 -19346 39233
-53671 33456 72631 92878
-21488 -72356 82223 -49350
-18732 -51822 86815 -4779
61202 -45012 91230 86227
-7809 -99306 44510 -65895
-98285 -39319 64258 -17226
-64449 -49812 -26138 63867
-33928 6963 1427 83195
-86188 -56695 -23892 -18261
34259 37599 83213 38315
-40232 -2954 -6576 30797
10524 -54715 35321 31145
-21572 -39613 75525 -21111
88224 -21542 88891 92358
35207 -95311 59759 -60532
-43329 -96887 -33161 12647
6530 33966 98432 57738
-22410 53317 63846 68528
39901 -55917 48394 52923
27807 -89637 69168 -71119
-35897 -52869 -33964 -29264
-88834 -67496 -23723 40562
1440 13683 67687 81860
37157 23338 88785 90123
13450 11210 99985 78745
4348 74917 53053 83004
-33840 42924 -11588 49429
-50482 -71273 -2570 8256
-65470 1165 -26896 79562
-33329 -88595 18083 -64590
-37150 22298 -19199 67566
1071 -44848 28200 53199
-89796 -24684 -75382 -17419
-71950 -80289 -34923 20815
-38874 -78186 60949 -6182
-44943 48446 24170 59060
-33848 46279 42085 57383
82562 20712 91657 86855
-76429 -24602 -64261 55388
-79603 -6827 -14778 75247
12157 -9506 99451 89580
-70864 81879 1034 94955
36761 -63132 86213 11840
-50123 -28546 -3041 84647
-71984 17415 -61521 90504
-54269 32633 59033 88274
-95645 50457 66077 57220
-64789 33122 -54445 47352
-25290 23848 60949 56980
-92333 -62157 49734 -39551
-64516 30643 -24637 36775
3243 -59093 55680 24988
-75452 48425 68148 93724
3735 -60334 94302 15458
-65159 -95034 -24787 10324
-92766 5442 -47098 43199
-97677 49476 -64030 69030
-58892 -63959 29595 -18812
-63793 -15528 -19600 79018
-38776 -74014 -21207 66646
-36002 -78652 -6565 86067
31703 10617 93651 59996
24956 -56662 66392 71434
37113 -81162 50403 -63146
-79054 -87803 -41918 11420
-84975 -72151 33774 -66754
-79316 64247 83780 90586
-37242 -86241 -30648 36118
-16810 -61307 -272 26312
-23100 59361 3349 90707
130 -31451 7421 19032
-25584 -81862 1544 -70724
50279 22213 69554 47792
-7076 -20529 20667 -16086
-42491 20949 -22731 36501
71127 -59328 75704 -8907
-67486 21970 -39767 32315
-88431 7165 82230 22134
-1520 11826 22197 18247
55126 -27296 68393 19498
-51960 -4400 -44766 15963
22167 90752 92749 93797
48627 -30285 67100 588
-58924 9601 57207 91263
33713 -86215 57763 13385
-29981 -78629 -724 -19719
-96782 12620 20280 47986
59502 -5627 84107 40620
-94898 -3638 -57423 8255
-20509 35176 55323 77289
-86999 -79912 48683 -50400
29459 -46724 42910 -40934
-25938 31118 55528 71382
-89181 -79525 -37064 4763
32071 -9602 99843 43186
-90587 5000 -60666 61467
43446 97084 97066 98734
-69102 4636 21037 49598
-91264 -39018 -11907 34310
7805 -28804 30907 44953
-15176 -89402 -10502 -86777
-88140 27934 -35290 29145
51502 -90529 58474 -70391
-8392 83839 45695 88626
-8366 -99040 16201 -5685
-38746 -64576 42124 -8518
10289 40581 94346 72395
-94246 -36495 -67781 -25485
-25237 13968 69499 94963
-45872 12583 22954 14270
82293 -45100 97725 29116
-75956 -15490 -63474 94775
43817 -78623 49847 9285
4560 -13426 24456 5373
-1773 -75349 896 -47829
17798 -84158 45160 50585
-89407 -91385 -59738 11897
-80168 -54776 -56129 31861
-73569 52697 -64559 95407
-84208 -86769 -80857 7251
-79612 -69314 -31031 -64085
-69241 -41389 -56543 -1514
-75164 -82381 -60920 88459
-61521 -94477 -31939 12782
-88965 -52709 -55654 -11691
7760 -98960 70405 -76691
56558 -52974 62086 41542
45679 -76472 47552 84591
62085 -93520 74429 38556
24899 -77185 78428 48195
-26532 -54215 -11469 93712
-93025 -60106 735 -11250
-15539 15814 -12211 96995
15752 -89157 32075 93106
-95149 -28612 -67784 -20071
21078 -86185 69535 -8532
1652 -59421 94022 56068
9200 -91121 47103 49930
33251 -38842 38054 92986
-92416 18228 -42756 19762
-68379 -97800 -61679 -35524
-87189 -78569 -50847 61216
50415 -92290 74061 -35802
23112 -45463 40047 36391
-73418 -93944 -34090 -54860
23851 -58346 62002 16566
18386 -38780 40033 -31183
-82221 -4422 -69828 21729
-24033 -94769 40866 -89432
-45864 1652 10672 34355
-29196 -85480 -8902 87233
18104 -34916 73479 -28695
9152 -39762 82232 510
-15772 -53091 88072 -29498
1502 -64755 15554 77270
-73018 -52772 -29771 23556
-22642 86501 17932 94781
-1632 35562 -230 59037
11128 8350 66152 28760
-81300 -73999 83733 -34719
-78739 -37978 40587 -34331
-78920 79374 72381 96213
-21490 65943 7717 99390
-45970 -88610 28470 -65420
2316 36205 95230 56817
-79867 17790 -66505 93549
-27659 39611 36281 47395
-78646 69423 85695 77865
-87103 17983 23119 51491
-89380 38847 13045 74743
19682 -80294 76245 31192
-81861 39127 46884 76271
-92276 -45063 -70244 43679
-79949 -54683 -48224 -31250
-81454 43541 94377 91158
-14253 -20017 87562 -16320
-47671 -33544 -27379 28402
-74034 -62658 -44889 55740
-88324 -25577 -9803 -13748
15190 -75954 66358 21607
-52710 36603 -39012 84082
-81918 -25225 -2990 -25212
-90855 -17165 -2817 -13176
-68207 -6657 -27685 48608
47698 -78066 85672 22685
-14574 5295 67585 93385
-7567 44253 59746 98909
-43479 33335 20292 39464
-16358 36956 4270 46073
-42135 -34882 -22417 1200
64229 61587 73492 81604
-37682 -93240 -14416 -80751
-25445 -80420 74879 -7476
-56805 -65766 -56750 38115
-31351 -99023 14524 -40166
6643 -59877 18096 -43621
-55721 14839 50488 86081
-59403 12859 -7567 45200
-52066 -86689 11242 -68656
27773 -64674 67420 41702
-95663 9188 -92209 70394
-20831 8970 89694 16051
9023 10537 30203 87800
68390 -56675 88870 -15008
-66226 6020 -32928 52074
-938 -71274 26404 -51807
22597 36122 80625 82104
-50276 76429 97419 82313
-91602 -60186 24240 -17222
-56099 -29171 -41100 -9661
12414 -16819 78533 88000
21973 -88897 46580 59062
-92786 -67367 91474 -59142
-68184 35483 6872 43801
-62254 -58017 -60614 68941
1277 -47848 21235 -33207
32399 -69230 46537 -55651
-84788 -83634 22601 -56327
62853 -63618 73738 62232
-70821 77668 33265 78949
-94832 -50047 45198 -18575
-3850 6961 36391 90175
-98919 -28646 -40624 30852
43633 2726 49925 16124
71050 11271 95460 18952
-58067 -49940 -15203 21481
30722 -98678 96231 6199
-82825 27910 85524 42178
29863 50642 29941 81110
-97572 -44921 -65593 11602
-75495 -60269 11856 -53292
-27977 34549 -26270 90284
42684 -95337 47570 68867
-98461 -1982 -12484 95441
-49040 24776 63791 41896
-59833 31372 2492 68664
-42034 -23725 -10818 66057
45648 -85998 75982 -69077
-71009 24152 1746 93307
52382 -50130 69125 55378
-18179 -77045 12415 -10142
-95809 -5508 -32266 -61
-63445 -11880 -37371 59225
53702 66044 98885 96695
-71073 -196 -61378 95185
-32106 -37278 14328 -1034
-70213 -24656 -22862 76423
-61937 -32457 58014 -32244
53919 -94403 62673 32354
-86009 -97048 39361 -21262
-46371 20770 93244 88312
-44461 29340 87689 73603
35267 -4824 40815 47548
-15179 -74121 16383 -23716
57381 5406 85976 51484
8210 -99415 23688 78879
-58210 47906 48070 75957
-46518 -99258 -20763 -65094
-69 25 -67 26
-24 0 -23 331
58 42 60 43
29 0 30 62
-20 224 -18 225
-13 0 -12 465
75 40 77 41
54 0 55 319
59 261 61 262
30 0 31 589
18 222 20 223
12 0 13 395
4 161 6 162
4 0 5 323
-68 255
59 44
-19 458
76 231
60 568
19 362
5 302
