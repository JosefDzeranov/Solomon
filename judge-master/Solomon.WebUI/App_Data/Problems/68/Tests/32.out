YES
2
0 465 -1
-68 465 -1
YES
2
0 589 1
59 589 1
YES
2
0 465 -1
-19 465 -1
YES
2
0 589 1
76 589 1
YES
2
0 589 1
60 589 1
YES
2
0 395 1
19 395 1
YES
2
0 323 1
5 323 1
